rust/core-lib/src/view.rs

```diff
@@ -163,12 +146,10 @@ impl View {
             Gesture { line, col, ty } =>
                 self.do_gesture(text, line, col, ty),
             GotoLine { line } => self.goto_line(text, line),
-            FindNext { wrap_around, allow_same } =>
-                self.find_next(text, false,
-                               wrap_around.unwrap_or(false),
-                               allow_same.unwrap_or(false)),
+            FindNext { wrap_around, allow_same: _ } =>
+                self.find_next(text, false, wrap_around.unwrap_or(false)),
```

Comment:
not for this PR, but an observation: instead of having `wrap_around` be an `Option<bool>`, I would have serde give it a default value if it's missing in the RPC.